src/Altinn.Platform/Altinn.Platform.Storage/UnitTest/Altinn.Platform.Storage.UnitTest.csproj

```diff
@@ -10,8 +10,6 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.All" />
-    <PackageReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" Version="2.2.0" />
```

Comment:
Should this be version 3.0.0? 
https://www.nuget.org/packages/Microsoft.AspNetCore.Mvc.Testing

Reply:
I don't see a reason for upgrading this package now if there is no new functionality we are dependent on that has been implemented.

Reply:
Okey